lib/src/base-styles/_nx-lists.scss

```diff
@@ -20,101 +22,96 @@
   }
 }
 
-.nx-list__title {
-  color: $nx-text-color;
-  margin: 0;
-}
-
 .nx-list__item {
-  @include semi-bold();
-  @include nx-truncate-ellipsis();
+  @include container-horizontal;
   @include container-vertical;
 
-  border-top: 1px solid transparent;
-  border-bottom: $nx-border-row;
+  border-top: 1px solid #e9e9e9;
   box-sizing: border-box;
-  color: $nx-text-color;
+  color: $nx-text-color-light;
+  display: grid;
+  grid-template-columns: 85% auto;
+  grid-template-rows: auto auto;
   margin: 0;
-  padding: $nx-spacing-sm $nx-spacing-xs;
+  padding: $nx-spacing-md;
 
   // we use first of type because we only want to modify the first list item,
   // not alerts, errors, or empty states
   &:first-of-type {
-    border-top: $nx-border;
+    border-top: 1px solid #d1d6f0;
```

Comment:
Looking at zeplin I think this is supposed to be #f4f5f9

Reply:
> Looking at zeplin I think this is supposed to be #f4f5f9

<img width="1460" alt="Screen Shot 2020-09-18 at 11 44 52 AM" src="https://user-images.githubusercontent.com/1991443/93617785-6a348100-f9a4-11ea-8027-95b558d83257.png">

Reply:
> Looking at zeplin I think this is supposed to be #f4f5f9

I must've clicked on the wrong thing